tests/cypress/e2e/filters-data.frame.cy.js

```diff
@@ -0,0 +1,150 @@
+describe("data.frame filters", () => {
+  beforeEach(() => {
+    cy.visit("http://127.0.0.1:5555/");
+    cy.waitForStabilityAndCatchError("body");
+    cy.get(".cy-active-summary-table tbody tr").as("activeSummary");
+  });
+
+  it("should filter proper values for numeric filters", () => {
+    cy.get(".cy-filter-card").contains("numeric").as("numericCard");
+    cy.get("@numericCard").click();
+    cy.waitForStabilityAndCatchError("body");
+
+    cy.get("@numericCard")
+      .get(".cy-numeric-selection-inputs input:first")
```

Comment:
I'm thinking about parent-child naming convention for filters. In `FilterState` card can have a general class like `filter-card`, then `RangeFilterState$ui_inputs` will just have `.numeric-filter-inputs`. 
```suggestion
      .get(".filter-card > .numeric-filter > .cy-testable.filter-card input:first")
```

Let's sit down and design a complete wireframe for teal/teal.slice and let's think about classes/ids of each element